streaming/src/main/scala/org/apache/spark/streaming/dstream/DStream.scala

```diff
@@ -277,7 +277,7 @@ abstract class DStream[T: ClassTag] (
     logInfo(s"Slide time = $slideDuration")
     logInfo(s"Storage level = ${storageLevel.description}")
     logInfo(s"Checkpoint interval = $checkpointDuration")
-    logInfo(s"Remember duration = $rememberDuration")
+    logInfo(s"Remember interval = $rememberDuration")
```

Comment:
It's consistent with "Checkpoint interval" above and the general impression that it's described as "interval" while called "Duration" in the code (due to the type).